bin/src/hickory-dns.rs

```diff
@@ -818,6 +818,8 @@ impl ConfigMetrics {
                     if let ExternalStoreConfig::Forward(_) = store {
                         self.zones_forwarder.increment(1)
                     }
+                    #[cfg(not(feature = "resolver"))]
```

Comment:
What does this solve? It looks a little strange.

Reply:
This fixes an unused variable warning on `store` if the resolver feature is off.